mllib/src/main/scala/org/apache/spark/mllib/tree/loss/LogLoss.scala

```diff
@@ -20,6 +20,12 @@ package org.apache.spark.mllib.tree.loss
 import org.apache.spark.annotation.{DeveloperApi, Since}
 import org.apache.spark.mllib.util.MLUtils
 
+/**
+ * Trait for adding probability function for the gradient boosting algorithm.
```

Comment:
moved to Loss.scala.  Removed doc on class.  Added doc for computeProbability method.